backend/src/kitconcept/intranet/profiles/default/registry/kitconcept.intranet.controlpanel.IIntranetSettings.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<registry>
+
+  <records interface="kitconcept.intranet.controlpanel.IIntranetSettings"
+           prefix="intranet"
```

Comment:
This should really use `kitconcept.intranet` as the prefix; the point of the prefix is to avoid collisions with settings that come from other packages.